plugins/UFM_NDT_Plugin/build/config/ndt.conf

```diff
@@ -16,6 +16,8 @@ host_patterns: (^(SAT|DSM)\d+ ibp.*$)
 port_validation_sleep_interval=5
 # number of attempts to verify boundary port state - polling
 port_validation_number_of_attempts=5
+# Flag if to include information about nodes with duplicated node description in merger report
+check_duplicated_node_description=False
```

Comment:
don't we want to have this check by default?

Reply:
Eitan Zahavi said that we do not need this check at all (merger will work correct even with duplications BTW).
Alex Fok said that need to add this verification to report too. 
Any way this will not fail verification like duplicated GUIDs (it is critical and topoconfig in this case will not be created)